sktime_dl/deeplearning/fcn/_regressor.py

```diff
@@ -48,17 +44,16 @@ def __init__(self,
         :param model_name: string, the name of this model for printing and file writing purposes
         :param model_save_directory: string, if not None; location to save the trained keras model in hdf5 format
         '''
-        self.verbose = verbose
-        self.is_fitted = False
-
-        # calced in fit
-        self.input_shape = None
-        self.history = None
-
-        # predefined
         self.nb_epochs = nb_epochs
         self.batch_size = batch_size
-        self.callbacks = callbacks if callbacks is not None else []
+
+        self.callbacks = callbacks
+        self.random_seed = random_seed
+        self.verbose = verbose
+        self.model_name = model_name
+        self.model_save_directory = model_save_directory
+
+        self.is_fitted = False
```

Comment:
This will be something we have to replace/refactor once with the new BaseEstimator from sktime has been released, but happy to keep it now

Reply:
The is_fitted line? Ok.